pyccl/core.py

```diff
@@ -763,11 +763,8 @@ def compute_growth(self):
         status = lib.cosmology_compute_growth(self.cosmo, status)
         check(status, self)
 
-    def compute_linear_power(self):
-        """Compute the linear power spectrum."""
-        if self.has_linear_power:
-            return
-
+    def _compute_linear_power(self):
+        """Return the linear power spectrum."""
```

Comment:
why do we need these?

Reply:
oops, leftover from the other branch, have fixed it now.